frontend/src/js/components/Profile/ProfilePage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {Grid, Row, Col} from 'react-bootstrap'
+
+
+class ProfilePage extends React.Component {
+  render () {
+    return (
+      <Grid>
+        <Row>
+          <Col xs={1}></Col>
+          <Col xs={10}>
+            <h1> {this.props.username}</h1>
+            <p> gender: {this.props.gender}</p>
+            <p> age: {this.props.age}</p>
+            <p> profession: {this.props.profession}</p>
+            <ul> <h5>topics: </h5> {this.props.topics.map((topic)=>{
```

Comment:
For your HTML to be valid, only `li` elements may be direct children of `ul` elements. The `h5` should be outside the `ul` in any case, it's not a member of the list.